models/config.yaml

```diff
@@ -53,6 +53,12 @@ llama-65b-gptq-3bit:
 .*vicuna.*(1.1|1_1|1.3|1_3):
   mode: 'instruct'
   instruction_template: 'Vicuna-v1.1'
+.*vicuna.*(1.5|1_5):
+  mode: 'instruct'
+  instruction_template: 'Vicuna-v1.1'
+  truncation_length: 4096
+  n_ctx: 4096
+  rms_norm_eps: 1.0e-05  
```

Comment:
According this https://github.com/ggerganov/llama.cpp/blob/e59fcb2bc129881f4a269fee748fb38bce0a64de/examples/server/server.cpp#L655 all llama2 should have `rms_norm_eps: 1.0e-05` .  In fact I dunno why other llama2 has 5.0e-6 in config.
There are also reports on Huggingface that people are having trouble with model responds when is `5.0e-6`, (extremely high perplexity or trash responds), especially with Vicuna-1.5. 
And the official Vicuna-1.5 repository mentions that it works with 1.0e-05 as per the official configuration. https://huggingface.co/lmsys/vicuna-13b-v1.5/blob/main/config.json 

As for `n_ctx`, I added it because most people are likely to be confused about how to set `rope_scaling` (`compress_pos_emb`) if they decide to use the 8k model. Because starting with lamma2 it uses the default 4k context and `rope_scaling` should be calculated as `max_seq_len/4096`, but in the UI we have hints for llama1 (`max_seq_len/2048`).
But if you think otherwise, I/you can remove `n_ctx`.

Reply:
This value is the result of a perplexity test performed using llama.cpp: https://github.com/ggerganov/llama.cpp/pull/2384

The UI hint is indeed misleading, maybe it should say something like "max_seq_len/2048 for llama-1 and max_seq_len/4096 for llama-2". Adding this change and removing n_ctx would be a good solution.

Reply:
This is odd, I've also seen people reporting that MythoLogic-L2-13B-GGML has a problem with `5.0e-6` and people solved it with 1.0e-05. Also their official config has `1.0e-05` https://huggingface .co/Gryphe/MythoLogic-L2-13b/blob/main/config.json
I would prefer to leave the values that are specified in the official repository.

Reply:
This is just a llama-2 fine-tune. There is no reason to give it special treatment.

Reply:
So, should I replace it to `5.0e-6`? Because Vicuna has its own naming standard, `rms_norm_eps` needs to be set anyway.

Reply:
I just played around with the model and it seems to work correctly with this `5.0e-6` value too.